lib/mix/lib/mix/tasks/do.ex

```diff
@@ -31,8 +31,11 @@ defmodule Mix.Tasks.Do do
 
   defp gather_commands([head | rest], current, acc)
       when binary_part(head, byte_size(head), -1) == "," do
-    part    = binary_part(head, 0, byte_size(head) - 1)
-    current = Enum.reverse([part | current])
+    current =
+      case binary_part(head, 0, byte_size(head) - 1) do
+        ""   -> Enum.reverse(current)
```

Comment:
Personally I think we're trying to move away from this kind of alignment in Elixir as it brings a lot of version control pain: when changing stuff around this, you usually have to change all the lines to keep the alignment and end up with a bigger diff than you would need if you didn't align. I think especially in such small cases the alignment brings almost nothing to the code, so personally I'd get rid of it :)